code/game/gamemodes/changeling/changeling_powers.dm

```diff
@@ -962,3 +962,71 @@ var/list/datum/dna/hivemind_bank = list()
 
 	feedback_add_details("changeling_powers","ED")
 	return 1
+	
+mob/proc/changeling_extract_dna_sting()
+	set category = "Changeling"
+	set name = "Arm Blade (20)"
+	set desc="Transform your arm into a deadly blade. Use it again to retract the blade."
+	
+	genetic_damage = 10
+	req_human = 1
+	max_genetic_damage = 20
+	weapon_type = /obj/item/weapon/melee/arm_blade
+	weapon_name_simple = "blade"
+
+/obj/item/weapon/melee/arm_blade
+	name = "arm blade"
+	desc = "A grotesque blade made out of bone and flesh that cleaves through people as a hot knife through butter"
+	icon = 'icons/obj/weapons.dmi'
+	icon_state = "arm_blade"
+	item_state = "arm_blade"
+	flags = ABSTRACT | NODROP
```

Comment:
I'm not sure we have a NODROP flag, I was sure we used a different variable for this.